docs/DEEP_DIVE.md

```diff
@@ -42,15 +48,17 @@ scripts/benchmark.sh
 
 Take a look at the source code of these demos before you run them. Check out our screencast of these demos for much more detail (note that there have been a few changes to our types since the screencast).
 
-Let's turn up the log level some more (and your favorite album) so you can see everything the server does as you run these demos:
+Let's turn up the log level some more (and your favorite album), so you can see everything the server does as you run these demos:
+
+- Open `src/config.zig` in your editor and change `log_level` to `3` (debug).
 
-* Open `src/config.zig` in your editor and change `log_level` to `3` (debug).
+- Rebuild tigerbeetle using the new debug log level by running `zig/zig build -Dcpu=baseline -Drelease-safe && mv zig-out/bin/tigerbeetle .`
 
-* Start a single replica cluster:
+- Start a single replica cluster:
 Init:
 `./tigerbeetle init --cluster=1 --replica=0 --directory=.`
 Run:
-`./tigerbeetle start --cluster=1 --replica=0 --addresses=3001 --directory=. &`
+`./tigerbeetle start --cluster=1 --replica=0 --addresses=3001 --directory=. 2>&1 | grep -v vsr &`
```

Comment:
```suggestion
`./tigerbeetle start --cluster=1 --replica=0 --addresses=3001 --directory=. &`
```

Reply:
The log at the documented level becomes impractical with the flood of vsr messages